app/src/main/java/xyz/hisname/fireflyiii/ui/transaction/TransactionSeparatorAdapter.kt

```diff
@@ -73,10 +68,13 @@ class TransactionSeparatorAdapter(private val clickListener:(Transactions) -> Un
         transactionModel.let { separator ->
             when(separator){
                 is SplitSeparator.SeparatorItem -> {
+                    val viewHolder = holder as SplitSeparatorViewHolder
+                    val separatorBinding = viewHolder.view
                     separatorBinding.separatorDescription.text = separator.description
                 }
                 is SplitSeparator.TransactionItem -> {
                     val viewHolder = holder as TransactionViewHolder
+                    val binding = viewHolder.view
```

Comment:
Apparently the existing implementation won't create any problems here because the list-items don't have a defined height (https://stackoverflow.com/questions/63131286/android-recyclerview-doesnt-recycle-items). So, one view will be created for every item and no recycling will occur. But I think that it should be corrected anyway.